clar2wasm/src/words/hashing.rs

```diff
@@ -52,7 +60,8 @@ impl SimpleWord for Hash160 {
         arg_types: &[TypeSignature],
         _return_type: &TypeSignature,
     ) -> Result<(), GeneratorError> {
-        traverse_hash("hash160", 160, generator, builder, arg_types)
+        // work space from sha256
+        traverse_hash("hash160", 160, generator, builder, arg_types, 64 + 8 + 289)
```

Comment:
Can you pls clarify the usage of those magic numbers `64 + 8 + 289`? Add a comment should be enough.